broker-cli/commands/orderbook.js

```diff
@@ -178,5 +224,6 @@ module.exports = (program) => {
     .command('orderbook', 'View the order book for a specific market')
     .option('--market <marketName>', MARKET_NAME_HELP_STRING, validations.isMarketName, null, true)
     .option('--rpc-address [rpc-address]', RPC_ADDRESS_HELP_STRING, validations.isHost)
+    .option('--json', JSON_FORMAT_STRING, program.BOOLEAN)
```

Comment:
It doesn't make sense to have a json output of a streaming function. If using json, we should perhaps either assume, or require that the user use the non-streaming endpoint `GetOrderbook`